lib/decorators/placeholder.vue

```diff
@@ -164,7 +195,9 @@
 
   export default {
     data() {
-      return {};
+      return {
+        isCollapsed: false
```

Comment:
isn't this true initially?

Reply:
I was thinking about that, and I think I'd be fine with having it collapsed initially.

Reply:
u mean expanded? !isCollapsed  = expanded, no?